yoti-sdk-impl/src/test/java/com/yoti/api/client/shareurl/policy/SimpleDynamicPolicyBuilderTest.java

```diff
@@ -192,6 +193,48 @@ public void buildWithRememberMeFlags() {
         assertTrue(result.isWantedRememberMeOptional());
     }
 
+    @Test
+    public void buildWithSelfieAuthorisationEnabledThenDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(true)
+                .withSelfieAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), not(hasItem(1)));
```

Comment:
Is it possible to refer to the constants for the auth types here, instead of `1`/`2`?

Reply:
If we make the constants public rather than private, yes @echarrod . Again this is something we're seeing in the Node SDK where these weren't available to anything else originally so I'll speak to @davidgrayston tomorrow about it.

Reply:
As discussed, we could define constants for the test to make this more readable